app/views/docs/_intro_version.html.erb

```diff
@@ -0,0 +1,52 @@
+<p>From <a href="https://18f.gsa.gov/2015/10/13/open-source-micropurchasing/">"Open
+Source Micro-purchasing: An experiment in federal acquisition"</a>:
+
+<blockquote>18F is an open-source team. We currently have hundreds of
+publicly available repositories, with dozens under active
+development. We've had numerous contributions from colleagues within
+government, and contributions from members of the public. But in the
+next few weeks, we are going to run an experiment: we want to contract
+for contributions. And we want to do it the 18F way.
+
+Part of contracting the 18F way is ensuring that all systems are built
+modularly and with APIs as capable as the human interface. The
+micro-purchase platform itself is no exception. This means that all
+data and transactions that are accessible via the web UI can be
+accessed by software using the API.
+</blockquote></p>
+
+<p>This documentation for using the API assumes you have some
+experience with writing clients for RESTful APIs, although for some
+situations you can just use <a href="https://curl.haxx.se/">curl</a>
+or <a href="https://stedolan.github.io/jq/">jq</a> to get the
+information you need.</p>
+
+<h2 id="version">Current Version</h2>
+
+<p>This documentation is generated from
+the <a href="<%= @swagger.base_path %>/swagger.json">API's
+specification file</a>, written
+in <a href="http://swagger.io">Swagger</a>. This file can be used to
+validate API responses and to automatically generate client
+libraries. We have not verified that auto-generated clients function
+correctly, but
+please <a href="https://github.com/18F/micropurchase/issues/new)">file
+an issue</a> if you try it and it works (or doesn't).</p>
+
+<p>The current version of the API is <%= @swagger.info.version %></p>
+
+<% swagger_spec_url = "https://micropurchase.18f.gov" +
+@swagger.base_path + "/swagger.json" %>
+
+<p>The Swagger specification can be found
+at <a href="<%= swagger_spec_url %>"><%= swagger_spec_url %></a></p>
```

Comment:
It's ugly, but I basically wanted to put the full path to the swagger.json, but I don't really have it anywhere in the document